index.html

```diff
@@ -3,12 +3,57 @@
 <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="./styles/styles.css" />
     <title>Currency converter</title>
 </head>
 <body>
+    <header>
+        <h1>Welcome to New Currency Converter</h1>
+    </header>
     <main>
+        <form>
+            <fieldset class="frame">
+                <legend>Insert rates</legend>
+                <label class="lbl_form" for="base_curr">Base currency</label>
+                <input id="base_curr" type="text" maxlength="3" value="EUR">
+                <label class="lbl_form" for="new_curr">New currency</label>
+                <input id="new_curr" type="text" maxlength="3" >
+                <label class="lbl_form" for="curr_rate">Rate</label>
+                <input id="curr_rate" name="curr_rate"  placeholder="1" value="1">
 
+                <button type="submit" class="btn_rate" onclick='onRateSubmit(event)'>Add rate</button>
+            </fieldset>
+        </form>
+
+        <form>
+            <fieldset class="frame">
+                <legend>From</legend>
+            <label class="lbl_form" for="amount_input">Amount</label>
+            <input id="amount_input" type="number" width="25px" placeholder="1" value="1">
+            <label class="lbl_form"  for="curr_input">Choose a currency:</label>
+            <select class='select_curr' id="curr_input" name="curr_input">
+                <option value="EUR">EUR</option>
+            </select>
+            </fieldset>
+            <fieldset  class="frame">
+                <legend>To</legend>
+            <label class="lbl_form" for="amount_output">Amount</label>
+            <input id="amount_output" type="number" width="30px">
+            <label class="lbl_form" for="curr_output">Choose a currency:</label>
+            <select class='select_curr' id="curr_output" name="curr_output">
+
+                <!-- <option value="dkk">DKK - Danish Krone</option>
```

Comment:
In order to have the codebase as clean as possible delete all commented code please